### DISCUSSION TITLE
4.6.5-beta update

### DISCUSSION THREAD

**Discussion by 0xspade** [simple-login/app#1511] [Self-hosting question] [answered]
Hi @nguyenkims,

Thanks for the update in simplelogin self-hosted. I already update my instance but noticed that it doesn't have any changes. Does it have any new features or something that we might be aware of or just some minor fix for some bugs?

Thanks,

**Comment by nguyenkims** [comment, marked as answer]
@0xspade this version allows you to add the "root" domain used during the setup as a custom domain in SimpleLogin. This way you can create custom aliases with it without the suffix.

**Reply by 0xspade** [reply]
interesting @nguyenkims, Thanks again for this wonderful update. Looking forward :)

**Comment by mfld-pub** [comment]
Was 4.6.5-beta pulled ? I only see the familiar 4.6.2-beta branch.

**Reply by Yougoshatenshi** [reply]
Its this branch for now: https://github.com/simple-login/app/tree/new-self-host-version
